src/core/varlink-cgroup.c

```diff
@@ -615,5 +615,9 @@ int unit_cgroup_runtime_build_json(sd_json_variant **ret, const char *name, void
                         JSON_BUILD_PAIR_CALLBACK_NON_NULL("IOReadBytes", get_io_counter_build_json, u),
                         JSON_BUILD_PAIR_CALLBACK_NON_NULL("IOReadOperations", get_io_counter_build_json, u),
                         JSON_BUILD_PAIR_CALLBACK_NON_NULL("IOWriteBytes", get_io_counter_build_json, u),
-                        JSON_BUILD_PAIR_CALLBACK_NON_NULL("IOWriteOperations", get_io_counter_build_json, u));
+                        JSON_BUILD_PAIR_CALLBACK_NON_NULL("IOWriteOperations", get_io_counter_build_json, u),
+
+                        /* OOM */
+                        JSON_BUILD_PAIR_UNSIGNED_NON_ZERO("OOMKills", crt->oom_kill_last),
```

Comment:
why NON_ZERO? it seems to me we want to suppress this if the concept doesn't make sense, but here you'd also suppress it if the concept makes sense but it at zero. Hence I think you can just drop _NON_ZERO, given you already validated that crt is non-NULL here...